docs/docs/integrating-with-other-libraries.md

```diff
@@ -0,0 +1,270 @@
+
+[Try it on CodePen.](http://codepen.io/wacii/pen/OWZJMQ?editors=0010)
+
+## Integrating with Model Layers
+
+React has little opinion on how data is stored or updated and so React components can easily incorporate the model layer from other frameworks. Models may be consumed as is or extracted using containers for better separation of concerns. Both approaches will be demonstrated using Backbone.
+
+### Using Backbone Models in React components
+
+The simplest way to consume Backbone models and collections from a React component is to listen to the various change events and manually force an update.
+
+Components responsible for rendering models would listen to `'change'` events, while components responsible for rendering collections would listen for `'add'` and `'remove` events. In both cases, call `this.forceUpdate()` to rerender the component with the new data.
+
+In the following code, `ListComponent` renders a collection with `ItemComponent` responsible for rendering the individual models.
+
+```js
+class ItemComponent extends React.Component {
+  constructor(props) {
+    super(props);
+    this.rerender = () => this.forceUpdate();
+  }
+
+  componentDidMount() {
+    this.props.model.on('change', this.rerender);
+  }
+
+  componentWillUnmount() {
+    this.props.model.off('change', this.rerender);
+  }
+
+  render() {
+    return <li>{this.props.model.get('text')}</li>;
+  }
+}
+
+class ListComponent extends React.Component {
+  constructor(props) {
+    super(props);
+    this.rerender = () => this.forceUpdate();
+  }
+
+  componentDidMount() {
+    this.props.collection.on('add', 'remove', this.rerender);
+  }
+
+  componentWillUnmount() {
+    this.props.collection.off('add', 'remove', this.rerender);
+  }
+
+  render() {
+    return (
+      <ul>
+        {this.props.collection.map(model => (
+          <ItemComponent key={model.cid} model={model} />
+        ))}
+      </ul>
+    )
+  }
+}
+```
+
+[Try it on CodePen.](http://codepen.io/wacii/pen/oBdgoM?editors=0010)
+
+### Extracting data from Backbone models
+
+Alternatively, whenever a model changes, you can extract its attributes as plain data. The following is [a higher-order component (HOC)](/react/docs/higher-order-components.html) that extracts all attributes of a Backbone model into state, passing the data to the wrapped component.
+
+This way, only the HOC needs to know about Backbone model internals, and the components concerned with presenting data can focus on that.
+
+```js
+function backboneModelAdapter(Component) {
+  return class extends React.Component {
+    constructor(props) {
+      super(props);
+      this.state = { ...props.model.attributes };
```

Comment:
What if there's a different `model` coming into `props`? I think it's worth showing how to handle it in `componentWillReceiveProps` because it's a common mistake in integrations.